packages/opencensus-exporter-stackdriver/src/types.ts

```diff
@@ -103,7 +103,7 @@ export interface Distribution {
 }
 
 export interface Point {
-  interval: {endTime: string, startTime: string};
+  interval: {endTime: string, startTime?: string};
```

Comment:
```startTime``` should be optional (Must be present for cumulative metrics only).  More info: https://cloud-dot-devsite.googleplex.com/monitoring/api/ref_v3/rest/v3/TimeInterval